egs2/wsj0_2mix/enh1/README.md

```diff
@@ -91,3 +91,25 @@ config: conf/tuning/train_enh_dc_crn_mapping_snr.yaml
 |---|---|---|---|---|---|
 |enhanced_cv_min_8k|0.93|14.86|14.16|24.02|13.73|
 |enhanced_tt_min_8k|0.94|14.25|13.46|23.13|13.01|
+
+<!-- Generated by ./scripts/utils/show_enh_score.sh -->
+# RESULTS
+## Environments
+- date: `Thu Apr 14 09:47:05 UTC 2022`
+- python version: `3.8.12 (default, Oct 12 2021, 13:49:34)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.7a1`
+- pytorch version: `pytorch 1.10.1+cu111`
+- Git hash: `9dbe4179b866b994f6914ef52ea7483696d22760`
+  - Commit date: `Wed Mar 16 13:25:26 2022 +0000`
+
+
+## svoice (multi-layer Si-SNR loss)
+
+ - config: conf/tuning/train_enh_svoice.yaml
+ - Pretrained model: https://huggingface.co/Zhaoheng/svoice_wsj0_2mix
+
+|dataset|STOI|SAR|SDR|SIR|
+|---|---|---|---|---|
+|enhanced_cv_min_8k|0.97|21.44|20.98|32.21|
+|enhanced_tt_min_8k|0.98|21.41|20.96|32.27|
```

Comment:
Could you also add the SI-SNR results? I think they are already there after running stage 8. You could obtain the scores by running:
```bash
paste /path/to/espnet/egs2/wsj0_2mix/enh1/exp/your_exp_name/enhanced_cv_min_8k/scoring/SI_SNR_spk{1,2} | \
    awk 'BEGIN{sum=0}{n=0;score=0;for (i=2; i<=NF; i+=2){n+=1;score+=$i}; sum+=score/n}END{printf ("SI_SNR(cv)=%.2f\n",sum/NR)}'
```
and
```bash
paste /path/to/espnet/egs2/wsj0_2mix/enh1/exp/your_exp_name/enhanced_tt_min_8k/scoring/SI_SNR_spk{1,2} | \
    awk 'BEGIN{sum=0}{n=0;score=0;for (i=2; i<=NF; i+=2){n+=1;score+=$i}; sum+=score/n}END{printf ("SI_SNR(tt)=%.2f\n",sum/NR)}'
```

Reply:
I see. The [`SI_SNR`](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/enh1/enh.sh#L79)  protocol was not there yet when I forked the repo. Let me rerun the scoring script and add the results.